content/en/blog/2022-06-03-multi-env-flux.md

```diff
@@ -0,0 +1,234 @@
+
+---
+title: "Multiple Environments with Flux and Kluctl"
+linkTitle: "Multiple Environments with Flux and Kluctl"
+date: 2022-06-03
+author: Alexander Block (@codablock)
+---
+
+Most projects that have server-side components usually need to be deployed multiple times, at least if you don't want to
+break things for your users all the time. This means, that there is not only one single "prod" environment, but also something
+like "test" or "staging" environments. So usually, there is a minimum of two different environments.
+
+Each of these environments has a defined role to play. The "prod" environment is obviously the environment that is used
+in production. A "test" or "staging" environment is usually the one where new versions are deployed so that they can
+be tested before being promoted to "prod". A "dev" environment might be the one where even high-risk deployments
+are allowed where breaking everything is daily business. You could even have multiple dev environments, e.g. one per
+developer.
+
+In the Kubernetes world, it is perfectly viable and good practice to run multiple environments on a single cluster.
+Depending on the requirements of your project, you might want to separate prod from non-prod environments, but you
+could still have "prod" and "staging" on the same prod cluster while "test", "dev", "uat" live on a non-prod cluster.
```

Comment:
life / running / are deployed on a non-prod cluster.

Reply:
I changed it to: "but you
could still have "prod" and "staging" on the same cluster while "test", "dev", "uat" exist on a non-prod cluster".

Hope this sounds better.